libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -15,6 +15,7 @@ export function Workspace () {
   const workspaceCreateTemplateInput = useRef()
 
   useEffect(() => {
+    setCurrentWorkspace(localStorage.getItem('currentWorkspace') ? localStorage.getItem('currentWorkspace') : '')
```

Comment:
if `localStorage.getItem('currentWorkspace')` is null of undefined, what will the value of current workspace?